compose/auth_certs/generate.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/sh
+
+set -xe
+
+openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -out certificate.pem -keyout private.key
```

Comment:
It would be nice to use the `DIR="$(dirname "$(readlink -f "$0")")"` trick (like in `smoketest.bash` so that when this script is run it outputs the files into the same directory as the script, rather than into whatever the `pwd` of the shell was.

ie if I do `cd cryostat3 ; sh compose/auth_certs/generate.sh` the files currently get created in `cryostat3`, but with the `DIR` line they could be placed into `cryostat3/compose/auth_certs` automatically, without me having to `mv` them or `cd` into that directory first.